### ISSUE TITLE
Add general overview of CAPRKE2

### ISSUE THREAD

**Issue by alexander-demicev** [rancher/turtles-docs#118]

**Comment by dharmit** [comment]
@alexander-demicev can you help me understand the scope of this issue? AFAIU, the CAPRKE2 docs are available [here](https://caprke2.docs.rancher.com/00_introduction.html). So what is this issue particularly about? Is this about adding CAPRKE2 overview under the "Reference guide" --> "Providers" section on [turtles docs website](https://turtles.docs.rancher.com/)?

**Comment by alexander-demicev** [comment]
@dharmit yes, maybe a couple words about CAPRKE2 and a reference to the project docs
